pkg/kv/kvserver/rangefeed/catchup_scan.go

```diff
@@ -253,6 +255,22 @@ func (i *CatchUpSnapshot) CatchUpScan(
 			recreateIter := false
 			// recreateIter => now is initialized.
 			var now crtime.Mono
+
+			// Under race tests we want to recreate the iterator more often to
+			// increase test coverage of this code. Doing this on every iteration
+			// proved a bit slow.
+			forceRecreateUnderTest := false
+			if util.RaceEnabled {
+				// Hash the pointer of our iterator into the range [0, 4] and only
```

Comment:
nit: `[0,4)` or `[0,3]`

Optionally, remove one step that needs comprehending, and say that the probability is 1/4 (for a fixed `iter`).